source/shared/cpp/ObjectModel/TextSection.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <string>
+#include <vector>
+#include "Enums.h"
+
+namespace AdaptiveCards
+{
+    class TextSection
+    {
+    public:
+        TextSection();
+        TextSection(std::string text, TextSectionFormat format);
+        TextSection(std::string text, std::string originalText, TextSectionFormat format);
+
+        std::string GetText() const;
+        std::string GetOriginalText() const;
```

Comment:
do we need to make distinction between original text and text?
get text will return original text even if parsing validation fails, right?